Sources/Views/ALKMessageButtonCell.swift

```diff
@@ -72,6 +72,8 @@ open class ALKMyMessageButtonCell: ALKChatBaseCell<ALKMessageViewModel> {
                                                       padding: ViewPadding.messageViewPadding)
         if !isMessageEmpty {
             messageView.update(model: model)
+        } else if #available(iOS 17, *) {
+            messageView.update(model: model)
```

Comment:
@AbhijeetRanjan308  is this the only place we have to cover for iOS 17 ?

Reply:
NO I added it in two places